internal/api/verify.go

```diff
@@ -624,8 +619,8 @@ func (a *API) verifyUserAndToken(ctx context.Context, conn *storage.Connection,
 		isValid = isOtpValid(tokenHash, expectedToken, sentAt, config.Sms.OtpExp)
 	}
 
-	if !isValid || err != nil {
```

Comment:
we don't need to check for `err` here because it's already handled in the `phoneChangeVerification` & `smsVerification` case above